src/components/governance/ModalDeposit.vue

```diff
@@ -92,6 +94,26 @@ export default {
         if (balance) return parseFloat(balance.amount)
       }
       return 0
+    },
+    transactionData() {
```

Comment:
like here this is a copy of `cosmos.MsgDeposit`

Reply:
Many are very similar, but not all. So I maintained a separation. But would love to improve this.